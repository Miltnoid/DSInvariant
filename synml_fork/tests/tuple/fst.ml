type nat =
  | O
  | S of nat

let fst : nat * nat -> nat |>
 { (0, 1) => 0
 ; (2, 3) => 2
 } = ?
