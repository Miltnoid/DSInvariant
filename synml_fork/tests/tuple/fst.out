(* Generated Sun Mar  1 19:57:29 2015 *)
let fst : nat * nat -> nat =
  fun (t1:nat * nat) -> #1 t1
;;
